contracts/test-contract/.gitpod.Dockerfile

```diff
@@ -0,0 +1,17 @@
+### wasmd ###
+FROM cosmwasm/wasmd:v0.18.0 as wasmd
```

Comment:
Actualy this `.gitpod.Dockerfile` file was inherited from the cosmwasm template since we are using the github CI this file can be safely removed instead of manteining another CI infrastrcure.